concrete/blocks/express_form/controller.php

```diff
@@ -834,4 +844,30 @@ protected function getFormEntity()
         return $entityManager->getRepository(\Concrete\Core\Entity\Express\Form::class)
             ->findOneById($this->exFormID);
     }
+
+
+    /**
+     * duplicate express entity attached to block express form and update block type data
+     * @param $update boolean if we like to save block type data after duplication
+     * @return int cloned express form entity id
+     */
+    public function duplicateEntityExpress($update=false)
```

Comment:
Duplicate express form entity and update block form data.
This method is called when:

1. Express form block added from scrapbook is changed 
2. Duplicate Page that contain an express form block (**alias**) ( concrete/src/Page/Cloner.php - method  duplicateExpressEntitiesOnPageDuplicate